.travis.yml

```diff
@@ -24,6 +24,54 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 4"
   - rvm: 2.4.1
```

Comment:
Seems we should be using ruby 2.4.3

https://puppet.com/docs/puppet/5.3/about_agent.html

Reply:
I've updated PR to use 2.4.3.